doxygen/src/apis_schema.txt

```diff
@@ -171,11 +172,17 @@ by either implementing or using a set of functions, data types, and definitions.
   </tr>
   <tr>
     <td>Capiversion</td>
-    <td>Version of this component. Is a mandatory part of the component ID.  The version format is described in \ref VersionType "Version Type".
+    <td>Version of the api. Is a mandatory part of the api ID.  The version format is described in \ref VersionType "Version Type".
```

Comment:
same here - does not seem to be related to licenceSets, should probably be fixed in separate PR / commit.
Also applies to other lines in this commit